Lottery-Game-Chainlink-VRF.md

```diff
@@ -21,7 +21,7 @@ Let's goo 🚀
 
 ## Intro
 
-- Chainlink VRF's are oracles which used to generate random values.
+- Chainlink VRF's are oracles which are used to generate random values.
```

Comment:
Could you change this to `Chainlink VRFs` from `Chainlink VRF's`, thanks.

Reply:
I have made the above requested changes. Please review and merge them.